customize.html

```diff
@@ -169,7 +160,6 @@ <h3>Popular Texture Packs</h3>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=1583">HDX Textures</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=14132">PixelBOX Reloaded</a></li>
       <li><a href="https://forum.minetest.net/viewtopic.php?t=14289">PixelPerfection</a></li>
-      <li><a href="https://forum.minetest.net/viewtopic.php?t=9314">Summerfields</a></li>
```

Comment:
why remove this one? it still looks usable

Reply:
is has an unfree license